### DISCUSSION TITLE
Account Issue

### DISCUSSION THREAD

**Discussion by 4-abdo** [vercel/community#3960] [Account Issue] [answered]
### Question

Registration problem? Unable to login? Blocked account?

**Comment by 4-abdo** [comment]
i can't sign up

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble registration and login. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
